src/platform/landing-pages/header-footer-data.json

```diff
@@ -0,0 +1,321 @@
+{
```

Comment:
This is copied from `staging.va.gov` just so it wouldn't die on me when the page loaded up. Probably we want to not have this for local application development, but that change can come in another PR.